tools/common/schema/types.go

```diff
@@ -138,6 +138,9 @@ const (
 	// CLIFlagQuiet is the cli flag for quiet mode
 	CLIFlagQuiet = CLIOptQuiet + ", q"
 
+	// CLIFlagConnectAttributesTxIsolation controls the transaction isolation version for mysql
+	CLIFlagConnectAttributesTxIsolation = "tx-isolation"
```

Comment:
I think there are two flavors of this - tx_isolation and transaction_isolation.. one of them is deprecated in the recent mysql version. We should either support both or alternatively, simply accept key-value attributes to be passed along to the dsn. I prefer the latter but I am ok with convenience option for the frequently used ones (like tx_isolation) as long as we support both flavors